cypress/integration/dib-travel/company-settings/approval-process.spec.ts

```diff
@@ -0,0 +1,49 @@
+import { ApprovalProcess } from '../../../models';
+
+describe('Approval Process Suite', () => {
+  let aps: ApprovalProcess;
+
+  before(() => {
+    cy.fixture('company-settings/approval-process').then((approvalProcessFormData) => {
+      aps = approvalProcessFormData;
+    });
+  });
+
+  beforeEach(() => {
+    cy.login();
```

Comment:
Your general use `beforeEach` should be something like:
```
cy.login();
cy.visit('/company-management/approval-process');
```

And testing only sidebar link (and confirming loaded content) should be a separate single test.